website2/docs/compiling-docker.md

```diff
@@ -130,8 +140,8 @@ After the commands, a new docker container is started with all the libraries and
 installed. The operation system is Ubuntu 18.04 by default. Now you can build Heron
```

Comment:
Should this line mention Debian instead of Ubuntu?

Reply:
Thanks for catching.  I reverted the script back ubuntu 18.04